servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy

```diff
@@ -161,6 +161,14 @@ final class ServiceTalkLibraryPlugin extends ServiceTalkCorePlugin {
   private static void configureTests(Project project) {
     project.configure(project) {
       test {
+        useJUnitPlatform()
+        // expected format for timeout: <number>[ns|μs|ms|s|m|h|d])
+        def junit5DefaultTimeout = System.getProperty("CI") ? "30s" : "10s"
```

Comment:
Not sure `System.getProperty("CI")` will give us what we need, should we use `Boolean.getBoolean(sysPropName)`?

Reply:
Actually, the `CI` is not a system property, it's env variable.

Reply:
https://github.com/apple/servicetalk/blob/35b4293f59279aba9fc7831d32819a1b414ad38d/docker/docker-compose.yaml#L35-L36

Reply:
using `System.getenv("CI")` instead

Reply:
perhaps `def junit5DefaultTimeout = Boolean.valueOf(System.getenv("CI") ?: "false") ? "30s" : "10s"`

Reply:
Yes, that looks better